infra/commit/signed-off-by-checker.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {readFileSync, writeFileSync} from 'fs';
+
+/**
+ * Argument List
+ *  - Path for Commit Message File
+ */
+var args = process.argv.slice(2)
+const commitMsgFilePath = args[0];
+
+// "ONE-vscode-DCO-1.0-Signed-off-by" should be found at least once.
+var hasSignedOffBy = false;
+var messages = readFileSync(commitMsgFilePath, 'utf-8').split(/\r\n|\r|\n/);
+for (let lineNum = 0; lineNum < messages.length; ++lineNum) {
+  if (messages[lineNum].includes('ONE-vscode-DCO-1.0-Signed-off-by')) {
```

Comment:
How about picking up the version 1.0? I mean...
```ts
// global var
var DCO_major_ver=1
var DCO_minor_ver=0
let signoff_str='ONE-vscode-DCO-'+DCO_major_ver+'.'_DCO_minor_ver+'-Signed-off-by'
```

I'm not sure the code works well :-)

Reply:
The idea is not bad, but version seems not that important I think because... DCO version of ONE is 1.0 for 4 years 😅
Honestly, I don't know the version is really needed or not... 😄 
But thanks for suggestion!